plugin/core/signature_help.py

```diff
@@ -68,7 +68,11 @@ def render(self, view: sublime.View) -> str:
         formatted = []  # type: List[str]
         if self.has_multiple_signatures():
             formatted.append(self._render_intro())
-        formatted.extend(self._render_label(view, signature))
+        self._function_color = view.style_for_scope("entity.name.function.sighelp.lsp")["foreground"]
+        self._active_parameter_color = view.style_for_scope("variable.parameter.sighelp.active.lsp")["foreground"]
+        self._inactive_parameter_color = view.style_for_scope("variable.parameter.sighelp.lsp")["foreground"]
+        self._emphasize_active_parameter = self._active_parameter_color == self._inactive_parameter_color
```

Comment:
The optimization better be worth it because I guess it will make it more difficult to experiment with different styles. :)

Reply:
Never mind, this just caches it for the life of the signature popup which disappears on switch views anyway.